indexing-service/src/main/java/io/druid/indexing/overlord/config/RemoteTaskRunnerConfig.java

```diff
@@ -32,6 +32,10 @@
   private Period taskAssignmentTimeout = new Period("PT5M");
 
   @JsonProperty
+  @NotNull
+  private Period taskCleanupTimeout = new Period("PT5M");
```

Comment:
5 minutes might be a bit short, maybe 15min in case things are taking a while to recover, just to be conservative.